cmake/BuildConfiguration.cmake

```diff
@@ -31,9 +31,17 @@ option(BUILD_SHARED_LIBS "Build Shared Libraries" OFF)
 CMAKE_DEPENDENT_OPTION(OPENKIT_MONOLITHIC_SHARED_LIB "Build monolithic OpenKit DLL with dependencies included" ON
                        "BUILD_SHARED_LIBS" OFF)
 
-# When other libraries are using a shared version of runtime libraries, OpenKit also has to use one.
-CMAKE_DEPENDENT_OPTION(OPENKIT_FORCE_SHARED_CRT "Use shared (DLL) run-time lib even when OpenKit is built as static lib." ON
-                       "NOT BUILD_SHARED_LIBS" OFF)
+if(MSVC)
+	# When other libraries are using a shared version of runtime libraries, OpenKit also has to use one.
+	# This only needs to be configured correctly if and only if OpenKit is built as static library (NOT shared library)
+	CMAKE_DEPENDENT_OPTION(OPENKIT_FORCE_SHARED_CRT "Use shared (DLL) run-time lib even when OpenKit is built as static lib." ON
+						   "NOT BUILD_SHARED_LIBS" OFF)
+
+	# Link C-Runtime static, even when building as dll
+	# This only has effect on windows platforms
+	CMAKE_DEPENDENT_OPTION(OPENKIT_FORCE_STATIC_CRT "Use static run-time lib even when OpenKit is built as dynamic lib." OFF
```

Comment:
shouldnt it be 'ON "BUILD_SHARED_LIBS" OFF' here, in the current code this doesnt have any effect

Reply:
The default value can be anything, as it only has effect when building DLLs.

I just use the dependent option for GUI here.